src/tools/dartpad/index.md

```diff
@@ -29,92 +28,81 @@ DartPad supports `dart:*` [core libraries](/guides/libraries) marked as
+  
 
-</ol>
+2. Choose a Flutter sample such as **Sunflower**, 
+   using the **Samples** list at the upper right.  
+   
+   The rendered output appears to the right.
 
 
 ### Create a command-line app {#step-2-server}
 
 To create a simple command-line app, use **New Pad**.
 
-<ol markdown="1">
-  <li markdown="1">
-  Click the **New Pad** button,
-  and confirm that you want to discard changes to the current pad.
-
-  </li>
-
-  <li markdown="1">
-  Click the Dart logo, make sure that **HTML** support is disabled,
-  and then click **Create**.
-  </li>
-
-  <li markdown="1">
-  Change the code. For example, change the `main()` function
-  to contain this code:
-
-<!-- library-tour/string-tests/bin/main.dart -->
-{% prettify dart tag=pre+code %}
-for (var char in 'hello'.split('')) {
-  print(char);
-}
-{% endprettify %}
-
-  As you type, DartPad shows hints, documentation,
-  and autocomplete suggestions.
-  </li>
-
-  <li markdown="1">
-  Click the **Format** button.
-  DartPad uses the [Dart formatter](https://github.com/dart-lang/dart_style#readme)
-  to ensure that your code has proper indentation, white space, and line wrapping.
-  </li>
-
-  <li markdown="1">
-  Run your app.
-  </li>
-
-  <li markdown="1">
-  If you didn't happen to have any bugs while you were entering the code,
-  try introducing a bug.
-
-  For example, if you change `split` to `spit`,
-  you get warnings at the bottom right of the window.
-  If you run the app, a compilation error appears in the console.
-  </li>
-</ol>
+1. Click the **New Pad** button,
+   and confirm that you want to discard changes to the current pad.
+   
+
+2. Click the Dart logo, make sure that **HTML** support is disabled,
+   and then click **Create**.
+   
+
+3. Change the code. For example, change the `main()` function
+   to contain this code:  
+   
+   <!-- library-tour/string-tests/bin/main.dart -->
+   {% prettify dart tag=pre+code %}
+   for (var char in 'hello'.split('')) {
+     print(char);
+   }
+   {% endprettify %}  
+   
+   As you type, DartPad shows hints, documentation,
+   and autocomplete suggestions.
+   
+
+4. Click the **Format** button.  
+   
+   DartPad uses the [Dart formatter](/tools/dart-format)
+   to ensure that your code has proper indentation, white space,
+   and line wrapping.
+   
+
+5. Run your app.
+   
+
+6. If you didn't happen to have any bugs while you were entering the code,
+   try introducing a bug.  
+
+   For example, if you change `split` to `spit`,
+   you get warnings at the bottom right of the window.
+   If you run the app, a compilation error appears in the console.
 
 
 ## Checking Dart version info
 
 The language features and APIs that DartPad supports depend on the
-**Dart SDK** version that DartPad is based on.
-You can find the SDK version at the bottom right of DartPad.
+**Dart SDK** version that DartPad is currently using.
+You can find the active SDK's version at the bottom right of DartPad.
```

Comment:
```suggestion
You can find this SDK version at the bottom right of DartPad.
```

(I'm not sure that "active SDK" is clear enough, or that we need to introduce "active SDK" as a term.)

Reply:
Thanks for pointing this out, it definitely seems unnecessary.